src/server/game/Groups/Group.cpp

```diff
@@ -1875,6 +1875,10 @@ GroupJoinBattlegroundResult Group::CanJoinBattlegroundQueue(Battleground const*
         // don't let join to bg queue random if someone from the group is already in bg queue
         if (bgTemplate->GetBgTypeID() == BATTLEGROUND_RB && member->InBattlegroundQueue())
             return ERR_IN_NON_RANDOM_BG;
+
+        // don't let join starter dk when they're not allowed to get teleported
```

Comment:
please reword: `don't let Death Knights join BG queues when they are not allowed to be teleported yet`

Reply:
done :)